mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -56,6 +56,11 @@ var (
 				Computed: true,
 				Description: `Whether Kubernetes master is accessible via Google Compute Engine Public IPs.`,
 			},
+			"private_endpoint_enforcement_enabled": {
```

Comment:
This will default to `false` (or null which means `false`), right?

Reply:
Yes, I've updated the descriptions.